src/tabs/docs/readme.md

```diff
@@ -33,7 +33,7 @@ AngularJS version of the tabs directive.
 
 * `deselect()`
   <small class="badge">$</small> -
-  An optional expression called when tab is deactivated. Supports $event in template for expression. You may call `$event.preventDefault()` in this event handler to prevent a tab change from occurring.
+  An optional expression called when tab is deactivated. Supports $event and selectedTabIndex in template for expression. You may call `$event.preventDefault()` in this event handler to prevent a tab change from occurring. The selectedTabIndex can be used to determine which tab was attempted to be opened.
```

Comment:
`selectedTabIndex` (backticks) - also could you similarly wrap $event while you're in there?  Thanks.